test/integration/data-types.test.js

```diff
@@ -277,6 +278,16 @@ describe(Support.getTestDialectTeaser('DataTypes'), function() {
   it('should parse an empty GEOMETRY field', function () {
    var Type = new Sequelize.GEOMETRY();
 
+   // MySQL 5.7 or above doesn't support POINT EMPTY
+   if (dialect === 'mysql' && semver.gte(current.options.databaseVersion, '5.7.0')) {
+     return;
+   }
+
+   // Postgres 9.5.0 or above doesn't support POINT EMPTY
+   if (/^postgres/.test(dialect) && semver.gte(current.options.databaseVersion, '9.5.0')) {
```

Comment:
Yes, absolutely :)

I guess how I was looking at it was that this PR opens up the possibility to test against multiple versions of dbs (mysql 5.5-5.7 and postres 9.x-9.5). The problem is that postgis 2.2 can be installed on any version of postgres 9.x. I was just observing that this is a test against the postgres version for a postgis test case and wondering if it would be better to test against the postgis version instead if that is even possible/available?